src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,132 @@
+package duke;
+
+import java.util.Scanner;
+
+import duke.task.Task;
+
+/**
+ * Represents the user interface
+ */
+public class Ui {
+
+    private static final String GREETINGS = "Hello! I'm Duke\n" + "What can I do for you?";
```

Comment:
Perhaps appending "_MESSAGE" here would be more in line with the coding standard, which specifies associated constants should have a common prefix. Maybe shifting the "MESSAGE" keyword itself to be a prefix instead of a suffix would be better too.